chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
@mjura @rhafer I am fine with moving the data to some directory like /var/lib/pgsql/data.old instead of deleting it and then deleting it after successful restore. Any thoughts?